packages/crawler/src/apify/apify-resource-creator.ts

```diff
@@ -40,6 +43,18 @@ export class ApifyResourceCreator implements ResourceCreator {
         }
     }
 
+    private async addUrlsFromPageCrawl(requestQueue: Apify.RequestQueue, page?: Page, discoveryPatterns?: string[]): Promise<void> {
+        if (page === undefined || discoveryPatterns === undefined) {
+            return Promise.resolve();
```

Comment:
You should be able to just call return here without specifically calling Promise.resolve.